calibrations/calibration_50-0536911040.yaml

```diff
@@ -0,0 +1,21 @@
+# Calibration file for the spare mobileye camera at STOL
```

Comment:
It looks like a calibration file from mako camera, not mobileye?

Reply:
Oops yeah that's a typo